functions.py

```diff
@@ -2,159 +2,159 @@
 from PIL import Image
 import os, sys
 
+class ImageResizer:
+    def cut_image(self):
+        print '''
+        =====================
+        CUT IMAGE
+        =====================
+        1. File
+        2. Folder
+        3. Close
+        =====================
+        '''
+        choice = raw_input(">> ")
+        if choice == "1":
+            directory = raw_input('Directory to file: ')
+            width = input('Width: ')
+            height = input('Height: ')
+            name = raw_input('Save as: ')
```

Comment:
Input handling should be extracted to `Main` class. Here we should just do operations on the images not read the input.